src/helpers.php

```diff
@@ -73,6 +73,24 @@ function config_path(string $path = ''): string
     return app('path.config') . (empty($path) ? '' : DIRECTORY_SEPARATOR . $path);
 }
 
+/**
+ * Get a password from an environment variable. If an environment variable
+ * called `${var}_FILE` is set, read the password from that file. Otherwise
+ * returns the content of the `$var` environment variable.
+ */
+function env_password(string $var): string|null {
+    $filename = env("{$var}_FILE", null);
+    if ($filename) {
+        if (file_exists($filename)) {
+            return file_get_contents($filename);
+        } else {
+            echo "The password file $filename does not exist. Please fix your config.";
```

Comment:
This line is not covered by unit tests, throwing a `RuntimeException` would be the better (testable) way with the same outcome.

Reply:
(Or eventually better: Just ignore the file if it can't be read)

Btw, using variables in `"` strings is not allowed according to our linter && it would be helpful to still being able to set a default value for `env(...)`